microprofile/tests/tck/tck-config/pom.xml

```diff
@@ -58,25 +58,99 @@
     </dependencies>
 
     <build>
+        <pluginManagement>
```

Comment:
This is part of my PR about build of Helidon on Windows. You have probably forgot to remove it when we were testing it.